Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarksViewController.swift

```diff
@@ -636,6 +592,83 @@ class BookmarksViewController: SiteTableViewController, ToolbarUrlActionsProtoco
             return fetchedBookmarkItem?.canBeDeleted ?? false
         }
     }
+    
+    func tableView(_ tableView: UITableView, contextMenuConfigurationForRowAt indexPath: IndexPath, point: CGPoint) -> UIContextMenuConfiguration? {
+        guard let bookmarkItem = fetchBookmarkItem(at: indexPath) else {
+            return nil
+        }
+        
+        var actionItemsMenu: UIMenu
+        
+        if bookmarkItem.isFolder {
+            var actionChildren: [UIAction] = []
+
+            let children = bookmarkManager.getChildren(forFolder: bookmarkItem, includeFolders: false) ?? []
+            
+            let urls: [URL] = children.compactMap { bookmark in
```

Comment:
Will this be an issue if there's 1000+ bookmarks?

Reply:
Existing feature for long time, we only moved it here to Menu actions.